playbook.yml

```diff
@@ -0,0 +1,34 @@
+---
+- hosts: all
+  gather_facts: False
+  become: True
+  pre_tasks:
+
+    # Check Ubuntu release version to determine if we need to install python 2,
+    # an Ansible dependency that isn't included by default in Ubuntu 16.04 and
+    # up.
+
+    - name: Check Ubuntu release
+      raw: cat /etc/lsb-release | grep DISTRIB_RELEASE | cut -d "=" -f2
+      register: ubuntu_release
```

Comment:
Addressed in [ad822ca](https://github.com/azavea/ansible-ntp/pull/2/commits/ad822cab8bb8c333bf5701157ecb064d653bcfab).